build.gradle

```diff
@@ -1010,6 +1015,53 @@ project(':core') {
   }
 }
 
+project(':metadata') {
+  archivesBaseName = "kafka-metadata"
+
+  dependencies {
+    compile project(':clients')
+    compile libs.jacksonDatabind
+    compile libs.jacksonJDK8Datatypes
+    compileOnly libs.log4j
+    testCompile libs.junitJupiter
+    testCompile libs.hamcrest
```

Comment:
Do we actually use hamcrest in this module? If not, we should remove this.

Reply:
yes, we use it